pkg/pcr/flow_detection.go

```diff
@@ -243,15 +243,24 @@ func isCBnT(fitEntries []fit.Entry) (bool, error) {
 	for _, fitEntry := range fitEntries {
 		switch fitEntry := fitEntry.(type) {
 		case *fit.EntryKeyManifestRecord:
-			data, data2, err := fitEntry.ParseData()
-			if data == nil && data2 == nil {
+			_, data2, err := fitEntry.ParseData()
```

Comment:
Can we just have "data" instead of "data2" now there's only a single data variable? Likewise below.

Reply:
I changed the naming, but not exactly as requested, so let me know if still needs improving :)